scipy/stats/tests/test_continuous_basic.py

```diff
@@ -231,9 +231,8 @@ def test_rvs_broadcast(dist, shape_args):
     # implementation detail of the distribution, not a requirement.  If
     # the implementation the rvs() method of a distribution changes, this
     # test might also have to be changed.
-    shape_only = dist in ['betaprime', 'dgamma', 'exponnorm',
-                          'nct', 'dweibull', 'rice', 'levy_stable',
-                          'skewnorm']
+    shape_only = dist in ['betaprime', 'dgamma', 'exponnorm', 'norminvgauss',
```

Comment:
Why? Other dists are in this list IIRC because `rvs` is too slow, is this the case here?

Reply:
No, it is because to generate rvs, I needs to sample from `norm`and ìnvgauss`. According to the comment in `test_rvs_broadcast(dist, shape_args)`, this means the test does not work in that case:

`    # If shape_only is True, it means the _rvs method of the
    # distribution uses more than one random number to generate a random
    # variate.  That means the result of using rvs with broadcasting or
    # with a nontrivial size will not necessarily be the same as using the
    # numpy.vectorize'd version of rvs(), so we can only compare the shapes
    # of the results, not the values.
`